<?xml version="1.0" ?>
<launch>
	<node pkg="robot_joy" type="transform_joy" name="transform_joy" >
		<param name="axis_linear_x" value="1" type="int" />
		<param name="axis_angular" value="3" type="int" />
		<param name="linear_max" value="1.5" type="double" />
		<param name="angular_max" value="0.7" type="double" />
	</node>
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
    </node>
</launch>
